recipe-server/client/control_new/components/forms/FormItem.js

```diff
@@ -50,21 +55,40 @@ export default class FormItem extends React.Component {
     initialValue: null,
     name: null,
     rules: null,
+    trimWhitespace: false,
   };
 
+  trimValue(event) {
```

Comment:
After seeing the `WrappedFormItem` stuff in the tests, I realized this function could be made `static` since it would make testing easier, and doesn't actually do anything with the component instance.